### ISSUE TITLE
Show date of latest commit from projects

### ISSUE THREAD

**Issue by jonathan-s** [lukaszx0/queues.io#119]
Some of the projects are old and it would be good to show the latest commit of each project. That way you can quickly get an inkling of whether the project is still alive and thriving.

**Comment by JensRantil** [comment]
Nice idea. The site is static today. Would it be possible to make a CORS request to Github's API to get that date?

**Comment by jonathan-s** [comment]
Yeah, I think that should be possible. Using this api request to github for celery for instance: https://api.github.com/repos/celery/celery/commits in the first object of that json list you can see the date. 

@JensRantil Are you a maintainer of this project btw?

**Comment by JensRantil** [comment]
No, I'm not. Just someone keeping an eye on it to see new queues showing up and doing occasional comments in issues to clarify them or bring them forward.
